modules/tinymce/src/core/test/ts/browser/UndoManagerTest.ts

```diff
@@ -682,4 +683,25 @@ describe('browser.tinymce.core.UndoManagerTest', () => {
     });
   });
 
+  context('TINY-9222: Scroll to the cursor after undo or redo actions', () => {
```

Comment:
Jira numbers should not be used in a `context` block because the context is about a type of action not a specific Jira like an actual test (plus they could contain multiple tests for different Jiras). In all honesty I don't see a need for a `context` here, especially when it's the same as the `it` block text so it should probably be removed entirely.

Reply:
Removed the `context` block whatsoever [here](https://github.com/tinymce/tinymce/pull/8205/commits/d03dcd64f8958af59ee6b7984f043033afb135d8)